src/runner/provider/qti.js

```diff
@@ -80,6 +81,13 @@ var qtiItemRuntimeProvider = {
             try {
                 //render item html
                 elt.innerHTML = this._item.render({});
+
+                // apply RTL layout according to item language
+                const $item = $(elt).find('.qti-item');
+                const itemLang = $item.attr('lang');
+                const dir = $item.attr('dir');
+                const $itemBody = $(elt).find('.qti-itemBody');
+                $itemBody.attr('dir', locale.getLanguageDirection(itemLang) || dir);
```

Comment:
**nitpick:** While it looks ok at a first glance, the `dir` attribute should be read from the itemBody as well. And it should be forced from the language only if missing.

```suggestion
                const $item = $(elt).find('.qti-item');
                const $itemBody = $item.find('.qti-itemBody');
                const itemDir = $itemBody.attr('dir');
                if (!itemDir) {
                    const itemLang = $item.attr('lang');
                    $itemBody.attr('dir', locale.getLanguageDirection(itemLang));
                }
```